doc/cli.markdown

```diff
@@ -982,45 +982,46 @@ increase verbosity
 
 # SSH
 
-## ssh [uuid]
+## ssh &#60;applicationOrDevice&#62; [serviceName]
 
-Warning: 'balena ssh' requires an openssh-compatible client to be correctly
-installed in your shell environment. For more information (including Windows
-support) please check the README here: https://github.com/balena-io/balena-cli
+This command can be used to start a shell on a local or remote device.
 
-Use this command to get a shell into the running application container of
-your device.
+If a service name is not provided, a shell will be opened on the host OS.
+
+If an application name is provided, all online devices in the application
+will be presented, and the chosen device will then have a shell opened on
+in it's service container or host OS.
+
+For local devices, the ip address and .local domain name are supported.
 
 Examples:
+	balena ssh MyApp
+
+	balena ssh f49cefd
+	balena ssh f49cefd my-service
+	balena ssh f49cefd --port <port>
+
+	balena ssh 192.168.0.1 --verbose
+	balena ssh f49cefd.local my-service
 
-	$ balena ssh MyApp
-	$ balena ssh 7cf02a6
-	$ balena ssh 7cf02a6 --port 8080
-	$ balena ssh 7cf02a6 -v
-	$ balena ssh 7cf02a6 -s
-	$ balena ssh 7cf02a6 --noninteractive
+Warning: 'balena ssh' requires an openssh-compatible client to be correctly
+installed in your shell environment. For more information (including Windows
+support) please check the README here: https://github.com/balena-io/balena-cli
```

Comment:
Not sure if it's better to point straight to the Additional Dependencies sections in INSTALL.md, as the instructions were moved there:
https://github.com/balena-io/balena-cli/blob/master/INSTALL.md#additional-dependencies

And if not, perhaps point directly to README.md instead of the repo's landing page. People who are not used to GitHub get confused about all the source code folders being displayed at the top of the page, and they don't realise that the README contents are displayed by scrolling down.

By the way, talking of "including Windows support", if you have a chance please test the ssh commands on [Windows](https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/). Just yesterday Thodoris was raising it in support and this forum thread:
https://forums.balena.io/t/ssh-to-container-fails-with-sh-if-f-usr-bin-balena/7344

Reply:
I've moved the documentation link :+1:

Regarding the windows failure, I reproduced, and it suffers from the same problem. The issue as I see it is that we execute commands in a system-specific subshell, and in that subshell the bash escaping is used - which obviously doesn't work in windows. Working on a fix now. All of the other features work by default.